xbmc/video/VideoInfoScanner.cpp

```diff
@@ -472,10 +472,16 @@ namespace VIDEO
           mediaType = MediaTypeTvShow;
         else if (info2->Content() == CONTENT_MUSICVIDEOS)
           mediaType = MediaTypeMusicVideo;
-        CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
-          mediaType, pItem->GetPath(), 24145,
-          StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(), URIUtils::GetFileName(pItem->GetPath()).c_str()),
-          pItem->GetArt("thumb"), CURL::GetRedacted(pItem->GetPath()), EventLevel::Warning)));
+
+        if (info2->Content() == CONTENT_TVSHOWS)
+          CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
+            mediaType, pItem->GetPath(), 24145,
+            StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(),CURL::GetRedacted(pItem->GetPath())), EventLevel::Warning)));
```

Comment:
While you're at it you could use `CMediaTypes::GetLocalization(mediaType)` to get the proper localized version of the media type instead of the internal string. Same applies to the instance below.

Reply:
You mean instead of `mediaType.c_str()`? Would that also make the `if`, `else if` section above obsolete where we currently check for the `mediaType`?

Reply:
Yes instead of `mediaType.c_str()`. But it does not make the `if-else if` section above obsolte because `CMediaTypes::GetLocalization()` expects a `MediaType` and not `CONTENT_MOVIES` etc.

Reply:
done